### ISSUE TITLE
Usage Statement

### ISSUE THREAD

**Issue by luckielordie** [linkerd/linkerd-await#29]
It would be good to see a usage statement for this project that can answer the following questions.

- Why I need to use linkerd-await?
- How I deploy linkerd-await?
   - Is it deployed already by the Linkerd helm chart? How do I ensure it's activated? Why isn't the behaviour automatic?
   - Is there a helm chart for deploying linkerd-await? Should I be rolling my own or do we need one as part of this project?

I was brought here by the following closed issue linkerd/linkerd2#2833. However I don't feel like I know enough about the repository to implement anything more than disabling my linkerd injection on all my Cronjobs.

Thanks.

**Comment by adleong** [comment]
Thanks @luckielordie, I agree.  We could really use a docs page describing when to use the `config.linkerd.io/proxy-await` annotation vs when to use linkerd-await vs when to use neither.  I think @kleimkuhler best understands the subtleties of this stuff.  Kevin, do you have time to write something up?

**Comment by kleimkuhler** [comment]
@adleong Yep I'd be happy to write something up that will answer your questions @luckielordie!
